handbook/company/pricing-features-table.yml

```diff
@@ -603,7 +606,7 @@
           moreInfoUrl: https://docs.google.com/document/d/1WzMc8GJCRU6tTBb6gLsSTzFysqtXO8CtP2sXMPKgYSk/edit?disco=AAAA6xuVxGg
       waysToUse:
         - description: Monitor performance for automated queries.
-        - description: Monitor performance for live queries. Coming soon (2024-01-26) #Customer-blanco
+        - description: Monitor performance for live queries. Coming soon (2024-01-26) #Customer-blanco #467
```

Comment:
Mike: FYI Instead of the issue number in the comment you can link to the issue directly as the moreInfoUrl underneath the description.

Reply:
Wow, great point. This is done for all issue numbers.